src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,10 +10,13 @@
  */
 public class Person implements ReadOnlyPerson {
 
+    public static int nextSequenceNumber;
```

Comment:
You have not assigned a value to nextSequenceNumber. It needs to be initialized.